packages/itwin/tree-widget/src/components/trees/common/components/BaseTree.tsx

```diff
@@ -168,7 +168,7 @@ function BaseTreeRenderer({
 
   return (
     <div style={{ position: "relative", height, overflow: "hidden" }}>
-      <div style={{ overflow: "auto", height: "100%" }}>
+      <div className="tw-tree-renderer-container" style={{ overflow: "auto", height: "100%" }}>
```

Comment:
Maybe use `id` instead of css class that does not have any styling?

Reply:
Switched to `id`.